pages/learn/manage/device-statuses.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Device statuses
+excerpt: Determine the status and connectivity of a device
+---
+
+# Device statuses
+
+The application overview page shows the status of each device in the application.
+
+<img src="/img/common/main_dashboard/application_device_status.png" alt="Application device status" width="40%" >
+
+Each device can have one of the following statuses:
+
+| Status            | Description                                                                                  |
+|-------------------|----------------------------------------------------------------------------------------------|
+| Online            | The device is online and is connected to either the VPN or has recent API communications.    |
```

Comment:
The `either` here sounds a bit too exclusive as a word to me. 
How about adding a `or both` at the end?
On the other hand I'm not a native English speaker...

Reply:
Yes, we can just drop the either here as not correct.